specs/src/state_machine_modules_v4.md

```diff
@@ -0,0 +1,35 @@
+# State Machine Modules v4
+
+The modules used in app version 4 are:
+
+## `celestia-app` modules
+
+- [blob](https://github.com/celestiaorg/celestia-app/blob/main/x/blob/README.md)
+- [minfee](https://github.com/celestiaorg/celestia-app/blob/main/x/minfee/README.md)
+- [mint](https://github.com/celestiaorg/celestia-app/blob/main/x/mint/README.md)
+- [signal](https://github.com/celestiaorg/celestia-app/blob/main/x/signal/README.md)
+- [tokenfilter](https://github.com/celestiaorg/celestia-app/blob/main/x/tokenfilter/README.md)
+
+## `cosmos-sdk` modules
```

Comment:
the modules under here at pointing at v0.46.11 of sdk

Reply:
ece84c87cc1dca0c1ae28fb8d6e5180749dac751